test/e2e/baseline_test.go

```diff
@@ -819,7 +819,57 @@ func TestL4Telemetry(t *testing.T) {
 	})
 }
 
-func buildL4Query(src, dst echo.Instance) prometheus.Query {
+func TestLongConnL4Telemetry(t *testing.T) {
+	framework.NewTest(t).Run(func(tc framework.TestContext) {
+		for _, src := range apps.EnrolledToKmesh {
+			for _, dst := range apps.EnrolledToKmesh {
+				tc.NewSubTestf("from %q to %q", src.Config().Service, dst.Config().Service).Run(func(stc framework.TestContext) {
+					localDst := dst
+					localSrc := src
+					opt := echo.CallOptions{
+						Port:                    echo.Port{Name: "http"},
+						Scheme:                  scheme.HTTP,
+						Count:                   500,
+						Timeout:                 2 * 60 * time.Second,
+						Check:                   check.OK(),
+						HTTP:                    echo.HTTP{Path: "/?delay=3s"},
+						To:                      localDst,
+						NewConnectionPerRequest: false,
```

Comment:
This flag will only take effect for HTTP2/3 but not HTTP1.1, ref: https://github.com/istio/istio/blob/master/pkg/test/echo/server/forwarder/http.go#L133

Also metioned here: https://github.com/istio/istio/blob/master/tests/integration/ambient/baseline_test.go#L345

So use HTTP2 instead, https://github.com/istio/istio/blob/master/tests/integration/ambient/baseline_test.go#L346